staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -250,14 +250,14 @@ type ObjectMeta struct {
 	// +optional
 	ClusterName string `json:"clusterName,omitempty" protobuf:"bytes,15,opt,name=clusterName"`
 
-	// LastApplied is a map of workflow-id to last applied
+	// OwnerClaims is a map of workflow-id to last applied
 	// configuration. A workflow can be the user's name, a
 	// controller's name, or the name of a specific apply path like
 	// "ci-cd". It keeps track of the ownership of fields. The last
 	// applied configuration is always in the same version as the
 	// parent object. It's used to keep track of the intent of the
-	// workflow-id that submitted that configuration.
-	LastApplied map[string]string `json:"lastApplied,omitempty" protobuf:"bytes,17,rep,name=lastApplied"`
+	// workflow-id that submitted that configuration. TODO
```

Comment:
You should probably mention that a workflow ID has to use a single version so that maybe we need to have the version in the workflow ID?